modules/behavior/draw_way.js

```diff
@@ -73,6 +74,8 @@ export function behaviorDrawWay(context, wayId, index, mode, startGraph) {
     function move(datum) {
         context.surface().classed('nope-disabled', d3_event.altKey);
 
+    // add one more boolean at end, is activeLayer===mapId
+        var _activeLayer = _find( Hoot.layers.loadedLayers, function(a, b) { return a.activeLayer; });
```

Comment:
I think we need to make use of this here, making sure to only apply a target loc if active ids match.

(here in iD)[https://github.com/openstreetmap/iD/blob/master/modules/behavior/draw_way.js#L85]  a question is asked (is this target loc one that allows vertex).

Our question needs to be `does the datum's mapId equal the _activeLayer mapId`